glossary.rst

```diff
@@ -0,0 +1,46 @@
+.. _glossary:
+
+========
+Glossary
+========
+
+This page contains definitions for Flux-specific terms and concepts mentioned
+throughout the various guides and docs on our sites.
+
+------------------------------------------------------------------------------
+
+**Flux Broker**
```

Comment:
Should we have references to each of the terms so they can be linked (specifically) from other pages?